addons/point_of_sale/views/pos_ticket_view.xml

```diff
@@ -0,0 +1,135 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="portal_my_home_ticket" name="Request an invoice from a ticket" inherit_id="portal.portal_my_home" customize_show="True" priority="40">
+        <xpath expr="//div[hasclass('o_portal_docs')]" position="inside">
+            <t t-call="portal.portal_docs_entry">
+                <t t-set="title">Request an invoice from a ticket</t>
+                <t t-set="url" t-value="'/pos/ticket'"/>
+            </t>
+        </xpath>
+    </template>
+
+    <!-- Used when entering the ticket manually -->
+    <template id="ticket_entry" name="Ticket Management">
+        <t t-call="portal.portal_layout">
+            <t t-set="no_breadcrumbs" t-value="True"/>
+            <t t-set="my_details" t-value="True"/>
+            <div class="row justify-content-md-center">
+                <div class="col-12 col-md col-lg-6 mt-4">
+                    <h3>Invoicing Request</h3>
+                    <form method="post" target="_self" action="/pos/ticket/validate">
+                        <input type="hidden" name="csrf_token" t-att-value="request.csrf_token()"/>
+                        <div class="col-lg-12" t-if="error_message" >
+                            <div class="alert alert-danger" role="alert">
+                                <t t-out="error_message"/>
+                            </div>
+                        </div>
+                        <div>
+                            <label class="col-form-label" for="order_reference">Order reference</label> <sup class="btn-link p-2" style="cursor: pointer;" title="You can find the reference at the bottom of your ticket.">?</sup>
+                            <input type="text" name="order_reference" t-attf-class="form-control" t-att-value="order_reference" required="true" placeholder="00001-001-0001"/>
+                            <label class="col-form-label" for="ticket_amount">Ticket amount</label> <sup class="btn-link p-2" style="cursor: pointer;" title="The ticket amount helps ensuring the authenticity of the request.">?</sup>
```

Comment:
```suggestion
                            <label class="col-form-label" for="ticket_amount">Ticket amount</label> <sup class="btn-link p-2" title="The ticket amount helps ensuring the authenticity of the request.">?</sup>
```
With the pointer, I thought it will open a new tab for me if I click, but it did nothing. Can't we just remove this style so that it becomes more consistent with the '?' from the backend view.